website/docs/flink-quick-start-guide.md

```diff
@@ -63,21 +38,34 @@ export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 # Start the Flink standalone cluster
 ./bin/start-cluster.sh
 ```
-#### Step.3 start Flink SQL client
+### Start Flink SQL client
 
 Hudi supports packaged bundle jar for Flink, which should be loaded in the Flink SQL Client when it starts up.
 You can build the jar manually under path `hudi-source-dir/packaging/hudi-flink-bundle`(see [Build Flink Bundle Jar](/docs/syncing_metastore#install)), or download it from the
 [Apache Official Repository](https://repo.maven.apache.org/maven2/org/apache/hudi/).
 
-Now starts the SQL CLI:
+Now start the SQL CLI:
 
 ```bash
-# HADOOP_HOME is your hadoop root directory after unpack the binary package.
-export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
-
-./bin/sql-client.sh embedded -j .../hudi-flink1.1*-bundle-*.*.*.jar shell
+# For Flink versions: 1.13 - 1.17
+export FLINK_VERSION=1.17 
+export HUDI_VERSION=0.14.0
+wget https://repo1.maven.org/maven2/org/apache/hudi/hudi-flink${FLINK_VERSION}-bundle/${HUDI_VERSION}/hudi-flink${FLINK_VERSION}-bundle-${HUDI_VERSION}.jar -P $FLINK_HOME/lib/
+./bin/sql-client.sh embedded -j lib/hudi-flink${FLINK_VERSION}-bundle-${HUDI_VERSION}.jar shell
 ```
 
+
+### Flink Support Matrix
```

Comment:
@ad1happy2go  We can move this part to the top of the Setup (as the first subsection under Setup) and see if we can make it common instead of under Flink SQL tab. ?